src/assets/index.js

```diff
@@ -1 +1,3 @@
-//assets
\ No newline at end of file
+// import {Homeicon} from "./home.svg";
+// export  {Homeicon};
+export {default as Homeicon} from "./home.svg"
```

Comment:
i think this statement has no use, so make the file blank if not uses in other files